.github/workflows/lint.yml

```diff
@@ -14,43 +14,5 @@ jobs:
       - name: Install dependencies
         run: cd cl && pip install -r recap_email/requirements.txt && pip install -r tests/requirements.txt
 
-      - name: Black Code Formatter
-        uses: psf/black@stable
-
-      - name: Run flake8
-        uses: suo/flake8-github-action@releases/v1
-        with:
-          checkName: 'lint'   # NOTE: this needs to be the same as the job name
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
-      - name: isort Import Sorter
-        uses: isort/isort-action@v1
-
-      - name: pylint Error Checker
-        run: pylint --fail-under 9 -f colorized cl
-
       - name: mypy Static Type Checker
         run: mypy --ignore-missing-imports .
-
-  lint-report:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v4
-      - uses: actions/setup-python@v2
-        with:
-          python-version: "3.11"
-
-      - name: Install dependencies
-        run: cd cl && pip install -r recap_email/requirements.txt && pip install -r tests/requirements.txt
-
-      - name: pylint Generate Report
-        run: >
-          pylint --exit-zero --load-plugins=pylint_json2html -f jsonextended cl |
-          pylint-json2html -f jsonextended -o pylint-report.html
-
-      - name: Upload report
-        uses: actions/upload-artifact@v4
-        with:
-          name: pylint report
-          path: pylint-report.html
```

Comment:
Do we need to replace this with `pre-commit` so the new linting tools run in the workflow?

Reply:
Yes, missed before, added now!

Reply:
Great! It seems that linting action is failing now.